proto/Dockerfile

```diff
@@ -17,8 +20,13 @@ RUN set -eux; \
       golang \
       nodejs \
       npm \
+      unzip \
       ; \
     go install google.golang.org/protobuf/cmd/protoc-gen-go@${PROTOC_GEN_GO_VERSION}; \
-    go install google.golang.org/grpc/cmd/protoc-gen-go-grpc@${PROTOC_GEN_GO_GRPC_VERSION}
+    go install google.golang.org/grpc/cmd/protoc-gen-go-grpc@${PROTOC_GEN_GO_GRPC_VERSION}; \
+    go install github.com/pseudomuto/protoc-gen-doc/cmd/protoc-gen-doc@${PROTOC_GEN_DOC_VERSION}; \
+    curl -LO https://github.com/protocolbuffers/protobuf/releases/download/v${PROTOC_VERSION}/protoc-${PROTOC_VERSION}-linux-x86_64.zip; \
+    unzip protoc-${PROTOC_VERSION}-linux-x86_64.zip -d /usr/local; \
+    rm protoc-${PROTOC_VERSION}-linux-x86_64.zip
```

Comment:
We should not be downloading protoc here; it is already done as part of `github.com/pseudomuto/protoc-gen-doc`.